editaveis/solucaoeletronica.tex

```diff
@@ -0,0 +1,367 @@
+    As conexões entre a \emph{raspeberry} e o módulo NRF é visto na Figura \ref{fritizingnrf} e na Tabela \ref{tabelanrf}.
+    
+    \begin{figure}[H]
+    \centering
+    \includegraphics[scale=0.6]{fritzingnrf.pdf}
+    \caption{Esquemático entre a \emph{raspberry} e o módulo NRF24, mostrando as conexões de cada pino entre eles.}
+    \label{fritizingnrf}
+\end{figure}
+
+    
+  \begin{table}[h]
+ \centering
+% distancia entre a linha e o texto
+ {\renewcommand\arraystretch{1.25}
+ \caption{Conexão entre a \emph{raspberry} e o módulo NRF24 por pinagem.}
+ \begin{tabular}{ l l }
+  \cline{1-1}\cline{2-2}  
+    \multicolumn{1}{|p{3.850cm}|}{NRF24L01 \centering } &
+    \multicolumn{1}{p{4.217cm}|}{\emph{Raspberry Pi 3} \centering }
+  \\  
+  \cline{1-1}\cline{2-2}  
+    \multicolumn{1}{|p{3.850cm}|}{VCC \centering } &
+    \multicolumn{1}{p{4.217cm}|}{17 (3.3V) \centering }
+  \\  
+  \cline{1-1}\cline{2-2}  
+    \multicolumn{1}{|p{3.850cm}|}{GND \centering } &
+    \multicolumn{1}{p{4.217cm}|}{25  \centering }
+  \\  
+  \cline{1-1}\cline{2-2}  
+    \multicolumn{1}{|p{3.850cm}|}{CE \centering } &
+    \multicolumn{1}{p{4.217cm}|}{15 \centering }
+  \\  
+  \cline{1-1}\cline{2-2}  
+    \multicolumn{1}{|p{3.850cm}|}{CS \centering } &
+    \multicolumn{1}{p{4.217cm}|}{18 \centering }
+  \\  
+  \cline{1-1}\cline{2-2}  
+    \multicolumn{1}{|p{3.850cm}|}{SCK \centering } &
+    \multicolumn{1}{p{4.217cm}|}{23 \centering }
+  \\  
+  \cline{1-1}\cline{2-2}  
+    \multicolumn{1}{|p{3.850cm}|}{MOSI \centering } &
+    \multicolumn{1}{p{4.217cm}|}{19 \centering }
+  \\  
+  \cline{1-1}\cline{2-2}  
+    \multicolumn{1}{|p{3.850cm}|}{MISO \centering } &
+    \multicolumn{1}{p{4.217cm}|}{21 \centering }
+  \\  
+  \cline{1-1}\cline{2-2}  
+    \multicolumn{1}{|p{3.850cm}|}{IRQ \centering } &
+    \multicolumn{1}{p{4.217cm}|}{- \centering }
+  \\  
+  \hline
+\label{tabelanrf}
+ \end{tabular} }
+\end{table}  
+
+
+\subsection{\emph{Universal Synchronous Receiver/Transmitter - UART}}
+    Para realizar a comunicação entre o microcontrolador \emph{Raspberry Pi 3} e o rádio \emph{BladeRF}, e o controlador de carga foi utilizado o protocolo de comunicação UART, isso devido ao seu alto índice de mobilidade na transmissão dos dados de um módulo para o outro. Para conseguir isso o protocolo realiza a transmissão e recepção dos dados de forma serial e assíncrona, isso significa que não é necessário que os módulos aos quais irão se comunicar através do protocolo UART estejam sincronizados no mesmo ciclo de \emph{clock} e que tanto a recepção (RX) como a transmissão (TX) possuem somente um cabo para se comunicar com o outro módulo, além disso a mesma possibilita uma comunicação simultânea entre os dois módulos ou seja, uma transmissão full-duplex. Através dessa mobilidade acaba-se tornando uma boa opção para realizar a transferência de dados entre os módulos citados.
```

Comment:
full-duplex tem que estar em itálico e tem que decidir se tem hífen ou não, pois lá pra cima está sem e agora está com